packages/charting/src/components/BenchmarkBarChart/BenchmarkBarChart.style.ts

```diff
@@ -0,0 +1,37 @@
+import { IBenchmarkBarChartStyleProps, IBenchmarkBarChartStyle } from './BenchmarkBarChart.types';
+
+export const getBenchmarkBarChartStyles = (props: IBenchmarkBarChartStyleProps): IBenchmarkBarChartStyle => {
+  const { width, benchmarkColor } = props;
+
+  return {
+    root: {
+      width: width || 300,
+      fontFamily: 'Segoe UI',
+      fontSize: '13px'
+    },
+    chartHeader: {
+      margin: '5px 0'
+    },
+    chartTitle: {
+      float: 'left'
+    },
+    percentage: {
+      float: 'right'
+    },
+    chartBody: {
+      clear: 'both',
+      paddingTop: '3px'
+    },
+    triangle: {
+      width: '0',
+      height: '0',
+      borderLeft: '8px solid transparent',
+      borderRight: '8px solid transparent',
+      borderTop: '8px solid',
+      borderTopColor: benchmarkColor || '#00A5B0'
```

Comment:
see if you can get color from fabric palatte

Reply:
Yes please use a semanticColor from theme. This will make it significantly easier to get working in themed apps.

Reply:
I can't get this color '#00A5B0' from fabric palatte. Do we have to use the color in fabric palatte?

Reply:
Pass in theme and get color from fabric palatte.blue